app/lib/zeebe-api/zeebe-api.js

```diff
@@ -143,14 +142,14 @@ class ZeebeAPI {
       if (zeebeGrpcClient) {
 
         /** @type {Array<ZeebeResource>} */
-        const resources = this._getZeebeResources(resourceConfigs, tenantId);
+        const resources = this._getZeebeResources(resourceConfigs);
 
         let response;
 
         if (resources.length > 1) {
           this._log.debug('deploying resources', resources);
 
-          response = await zeebeGrpcClient.deployResources(resources, tenantId);
+          response = await zeebeGrpcClient.deployResources(resources);
```

Comment:
Great that we solve the tenant configuration in the client so that it's impossible to miss.